lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -891,6 +891,7 @@ function dosomething_reportback_get_reportback_total_by_nid($nid, $type = 'quant
 
   // Select all reportbacks for given $nid.
   $query = db_select('dosomething_reportback', 'rb')->condition('nid', $nid);
+  $query->condition('promoted', 0); // Prevents non-reviewed RB's from being added to total
```

Comment:
You should search for where it is null.
`$query->isNull('promoted')`

Also, code style, the comment should go above the line